### PR TITLE
feat: consolidating fee columns

### PR DESCRIPTION
**Pull request by aballerr**
https://uniswaplabs.atlassian.net/browse/WEB-1886
https://uniswaplabs.atlassian.net/browse/WEB-1887
https://uniswaplabs.atlassian.net/browse/WEB-1888
https://uniswaplabs.atlassian.net/browse/WEB-2511

1. add floor price column
2. added last price column
3. consolidated royalties and market fee + added hover state
4. I improved the space formatting.  Should be noted more work needs to be done on screen size compatibility.  I believe this is an improve experience over current prod

### CODE REVIEW

**Review by cbachmeier — CHANGES_REQUESTED** [review]
Found some minor bugs. But aligned with our tech debt strategy there are a lot of components here that should be converted to Styled components and static strings that should be internationalized

File: src/nft/components/profile/list/NFTListingsGrid.tsx [hunk 1]
```diff
@@ -398,36 +437,54 @@ const MarketplaceRow = ({
...
+        {asset.floorPrice ? `${asset.floorPrice.toFixed(2)} ETH` : '-'}
+      </Column>
+      <Column
+        className={bodySmall}
+        color="textSecondary"
+        textAlign="left"
+        flex="1"
+        display={{ sm: 'none', md: 'none', xl: 'flex' }}
+      >
+        {asset.lastPrice ? `${asset.lastPrice.toFixed(2)} ETH` : '-'}
```
Comment (cbachmeier) [inline src/nft/components/profile/list/NFTListingsGrid.tsx:457]: For most places we go to 3 decimals, can you check with design that 2 is intentional here. For example on Ikegai the floor we show on the collection page is 0.001 but on his list page it shows as 0.00. We also have some auto formatting functions in currency utils that handle trailing zeroes.
Screen Shot 2022-12-22 at 08 56 38

File: src/nft/components/profile/list/NFTListingsGrid.tsx [hunk 2]
```diff
@@ -398,36 +437,54 @@ const MarketplaceRow = ({
...
+            onMouseEnter={handleHover}
+            onMouseLeave={handleHover}
+            onClick={(e) => {
+              e.stopPropagation()
+              removeAssetMarketplace(asset, selectedMarkets[0])
+              removeMarket && removeMarket()
+            }}
+          >
+            <Box className={styles.removeMarketplace} visibility={hovered ? 'visible' : 'hidden'} position="absolute">
+              <Box as="img" width="32" src="/nft/svgs/minusCircle.svg" alt="Remove item" />
```
[the remark this answers is not in the document]
Comment (aballerr) [reply src/nft/components/profile/list/NFTListingsGrid.tsx:475]: This is an existing issue on prod when multiple markets are selected, but I will try and fix

**Review by cbachmeier — APPROVED** [review]
LGTM. Going to move updating styled components to the follow-up PR to prevent scope creep and get other eyes on the PR

**Pull request merged**